frontend/src/components/Options.vue

```diff
@@ -239,4 +387,102 @@ export default Vue.extend({
   height: 20px;
   filter: invert(75%) sepia(8%) saturate(243%) hue-rotate(8deg) brightness(96%) contrast(81%);
 }
-</style>
+.menu-burger {
+  font-size: 35px;
+}
+
+.options {
+  padding : 25px;
+}
+.burger-icon {
+  width: 35px;
+}
+.burger-icon:hover {
+  cursor: pointer;
+}
+
+.menu-open {
+    height: 100%;
+    width: 100%;
+    top: 0;
+    position: fixed;
+    z-index: 999999;
+    background-color: rgb(0 0 0 / 85%);
+    left: 0;
+    padding: 40px;
+}
+
+.x-button {
+  float: right;
+  color : white;
+  font-size: 23px;
+  margin-right: 60px;
+}
+.x-button:hover {
+  cursor: pointer;
+}
+.ads {
+  float: left;
+  width: 60%;
+  height: 100%;
+  margin-top: 30px;
+}
+.menu-icons {
+  float : right;
+  width: 40%;
+  height: 100%;
+  padding-top: 15px;
+  display: inline;
+}
+
+.menu-icon {
+  width: 95px;
+  height: 95px;
+  background: #1A253B 0% 0% no-repeat padding-box;
+  border: 1px solid #344362;
+  border-radius: 5px;
+  opacity: 1;
+  text-align: center;
+  padding-top:20px;
+  color : white;
```

Comment:
When you're copying css from the designs, try to trim unnecessary styles (like opacity: 1 for example)